### PR TITLE
Updating Information for 1.0.0a6 Release

### PR DESCRIPTION
**Pull request by emarinier**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,15 @@
...
 
-All notable changes to Proksee will be documented in this file. 
+All notable changes to Proksee will be documented in this file.
+
+v1.0.0a6
+
+- Modifies the Mash database download procedure to write less output.
+- Correctly instructs QUAST to use the user-specified minimum contig size.
+- Better handling of Mash for species estimation of large assemblies with many contigs.
+- Mash may now be run using parrellization.
```
Comment (apetkau) [inline CHANGELOG.md]: Change **parrellization** to **parallelization**

**Review by apetkau — CHANGES_REQUESTED** [review]
Looks good. Just one suggested change.

File: CHANGELOG.md (hunk 1 above)
Comment (emarinier) [reply CHANGELOG.md]: Thanks! I can't spell! Haha.

Comment (apetkau) [reply CHANGELOG.md]: Heh, nor can I 😛

**Review by apetkau — APPROVED** [review]
Looks good to me. Thanks so much.

**Pull request merged**